.github/scripts/prepare_data.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+# This script takes as input a directory, the name of the archive and a version.
```

Comment:
:art: It would be nice if it had a clear usage doc comment like

`Usage: prepare_data.sh <models_dir> <model_name> <version_tag>  [--usage]`

Reply:
There is an instantiated example a few lines below. I can make it more general